analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -48,6 +48,15 @@ import org.ossreviewtoolkit.spdx.toSpdx
 
 private const val DEFAULT_SCOPE_NAME = "default"
 
+private val SPDX_SCOPE_RELATIONSHIPS = setOf(
```

Comment:
Commit message: s/refactor/Refactor/

Reply:
As we hard-code the members here, it should be fine to use just `listOf()` instead of `setOf()`.

Reply:
I don't see any benefit in using `listOf()`. On the contrary, this clearly shows that there are no duplicate elements in it, which helps later code changes to not think unnecessary duplicate entry checks are necessary.